### PR TITLE
SDI-4: ✨ Add establishment filter and minor ui fixes

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

File: backend/controllers/prisonerProfile/prisonerVisits.ts [hunk 1]
```diff
@@ -11,34 +11,40 @@ export const VISIT_TYPES = [
 const calculateStatus = ({ cancelReasonDescription, completionStatusDescription, completionStatus, startTime }) => {
   switch (completionStatus) {
     case 'CANC':
-      return `Cancellation: ${cancelReasonDescription}`
+      return cancelReasonDescription ? `Cancelled: ${cancelReasonDescription}` : 'Cancelled'
```
Comment (petergphillips) [inline backend/controllers/prisonerProfile/prisonerVisits.ts:14]: cancel reason sometimes not populated

File: backend/controllers/prisonerProfile/prisonerVisits.ts [hunk 2]
```diff
@@ -80,7 +86,7 @@ export default ({ prisonApi, pageSize = 20 }) =>
                 status,
                 name: formatName(visitor.firstName, visitor.lastName),
                 age: `${visitor.dateOfBirth ? moment().diff(visitor.dateOfBirth, 'years') : 'Not entered'}`,
-                relationship: visitor.relationship,
+                relationship: visitor.relationship || 'Not entered',
```
Comment (petergphillips) [inline backend/controllers/prisonerProfile/prisonerVisits.ts:89]: relationship sometimes empty

File: backend/tests/prisonerVisits.test.ts [hunk 3]
```diff
@@ -5,11 +5,14 @@ describe('Prisoner visits', () => {
...
-  const prisonApi = {}
+  const prisonApi = {
+    getDetails: jest.fn(),
+    getVisitsPrisons: jest.fn(),
+    getVisitsForBookingWithVisitors: jest.fn(),
+  }
 
   let req
   let res
-  let logError
```
Comment (petergphillips) [inline backend/tests/prisonerVisits.test.ts:12]: not doing anything here

File: backend/tests/prisonerVisits.test.ts [hunk 4]
```diff
@@ -125,8 +130,8 @@ describe('Prisoner visits', () => {
               eventOutcomeDescription: 'Absence',
               visitTypeDescription: 'Official Visit',
               prison: 'Leeds (HMP)',
-              completionStatus: 'SCH',
-              completionStatusDescription: 'Scheduled',
+              completionStatus: 'CANC',
```
Comment (petergphillips) [inline backend/tests/prisonerVisits.test.ts:133]: test data change to test the display without a cancel reason

**Review by connormaglynn — APPROVED** [review]
LGTM

**Pull request merged**